gcp/modules/gcp-stackdriver-monitoring/resources/alert_policies/couchdb_prometheus_exporter_exports_metric.json

```diff
@@ -1,5 +1,5 @@
 {
-  "display_name": "couchdb_prometheus_exporter exports a metric",
+  "display_name": "CouchDB Prometheus Metric Exporter Check",
```

Comment:
I see that you are trying to normalize display_names, but I prefer the old name. I don't know what any of the "checks" do based on their names, but I think it's clear what this test is testing for.

Reply:
I reviewed our existing alert policies – most of the names are not self-explanatory. I'll open a separate ticket to address this and revert all naming changes in this PR.